internal/plugins/manifests/v2/init.go

```diff
@@ -74,6 +75,14 @@ func (s *initSubcommand) Scaffold(fs machinery.Filesystem) error {
 	// Append SDK recipes.
 	switch operatorType {
 	case projutil.OperatorTypeGo:
+		//(TODO): This removes `CGO_ENABLED=0` references from the go/v3 and go/v4-alpha scaffolds for FIPS compliance.
+		// This is a temporary workaround to remove CGO_ENABLED=0 references in the Go plugin chain from files scaffolded from upstream plugins.
+		// We would need to remove this when the fix is made in upstream Kubebuilder eventually.
+		err = util.ReplaceInFile("Dockerfile", "CGO_ENABLED=0", "")
+		if err != nil {
+			return err
+		}
```

Comment:
It feels wrong to me to put this functionality in the `manifests` plugin. Should we have a separate temporary plugin for this?

Reply:
Could you please elucidate what you mean by a separate temporary plugin? Would it be an overkill to have a temporary plugin just for this? 

Wondering if having that would eliminate the need to make changes in the Kubebuilder, and if so, that makes sense to me. But if we need to make changes in Kubebuilder anyway, do we still want to have a separate temporary plugin for this?